content/python/tags.py.md

```diff
@@ -0,0 +1,215 @@
+---
+title: "Working with Tags"
+description: "A few examples on interacting with Tags"
+
+date: "2020-02-19"
+classes: 
+    - "SoftLayer_Tags"    
+tags:
+    - "Tags"
+---
+
+[Tags](https://https://sldn.softlayer.com/reference/services/SoftLayer_Tags/)
+
+### Get all tags
+
+```python
+import SoftLayer
+import json
+
+client = SoftLayer.create_client_from_env()
+account_service = client['Account']
+
+try:
+    response = account_service.getTags()
```

Comment:
This is a great spot to show an example of the latest softlayer-python auto-iteration.

Add something like this.
===========


> Notes using [`iter_call`](https://softlayer-python.readthedocs.io/en/latest/api/client/#making-api-calls) requires softlayer-python v5.5.3+

```
response = client.iter_call('SoftLayer_Account', 'getTags', limit=50)
for tag in response:
    print(tag)
```
===========